src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig

```diff
@@ -1,55 +1,65 @@
+  * versions in the future. If you wish to customize PrestaShop for your
+  * needs please refer to http://www.prestashop.com for more information.
+  *
+  * @author    PrestaShop SA <contact@prestashop.com>
+  * @copyright 2007-2018 PrestaShop SA
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  * International Registered Trademark & Property of PrestaShop SA
+  *#}
 <div class="ps-dropdown dropdown btn-group bordered mb-1">
-    {% for menu in topMenuData %}
-        {% set refMenu = menu.refMenu %}
-        <div id="{{refMenu}}" class="dropdown-label" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            <span class="js-selected-item selected-item module-category-selector-label">
-                {{menu.name}}
-            </span>
-            <i class="material-icons arrow-down float-right">keyboard_arrow_down</i>
-        </div>
-        <div class="ps-dropdown-menu dropdown-menu module-category-selector" aria-labelledby="{{refMenu}}">
-            <ul class="items-list js-items-list">
-                <li class="module-category-reset">
-                    <a class="dropdown-item" href="#">
-                            {{ 'All Categories'|trans({}, 'Admin.Modules.Feature') }}
-                    </a>
-                </li>
-                {% for subMenu in menu.subMenu|arrayCast %}
-                    <li class="module-category-menu"
-                        {% if subMenu.tab %}data-category-tab="{{subMenu.tab}}"{% endif %}
-                        data-category-ref="{{subMenu.refMenu}}"
-                        data-category-display-name="{{subMenu.name}}"
-                    >
-                        <a  class="dropdown-item" href="#">
-                            {{subMenu.name}}<span class="float-right">{{subMenu.modules|length }}</span>
-                        </a>
-                    </li>
-                {% endfor %}
-            </ul>
-        </div>
-    {% endfor %}
+  {% for menu in topMenuData %}
+    {% set refMenu = menu.refMenu %}
+    <div id="{{refMenu}}" class="dropdown-label" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      <span class="js-selected-item selected-item module-category-selector-label">
+        {{ 'All Categories'|trans({}, 'Admin.Modules.Feature') }}
+      </span>
+      <i class="material-icons arrow-down float-right">keyboard_arrow_down</i>
+    </div>
+
+    <div class="ps-dropdown-menu dropdown-menu module-category-selector" aria-labelledby="{{refMenu}}">
+      <ul class="items-list js-items-list">
+        <li class="module-category-reset">
+          <a class="dropdown-item" href="#">
+            {{ 'All Categories'|trans({}, 'Admin.Modules.Feature') }}
+          </a>
+        </li>
+        <li class="module-category-menu module-category-recently-used"
+          data-category-ref="recently-used"
+          data-category-display-name="{{ 'Recently Used'|trans({}, 'Admin.Modules.Feature') }}"
```

Comment:
What is it supposed to mean?

Reply:
Okay, I get it. Why is there an upper case at the beginning of the second word?

Reply:
Same as Mock up :/ Don't understand why.

Reply:
Mathias used a capital, can you change to Recently used please ?